model/AI/Classes/EnergyConsumptionDescription

```diff
@@ -0,0 +1,27 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# EnergyConsumptionDescription
+
+## Summary
+
+The Class that helps note down the actual energy consumption value and the unit in which it is recorded.
```

Comment:
Remove word "actual" - see comments from @bact below.  I think that in some cases "actual" may be hard to establish - so best estimate may be all that is possible.